static/src/stylesheets/_vars.scss

```diff
@@ -120,6 +120,8 @@ $frontend-palette: (
 
     paidfor-background:     guss-colour(news-support-3, $pasteup-palette),
 
+    special-report-accent:  #fce800,
```

Comment:
Do these follow some other palette, then? Or do we have free reign to just choose whatever our accessibility tool recommends?